testdata/jsight_0.3/ser-to-openapi/req.jschema.example.number_0.2/number.openapi

```diff
@@ -0,0 +1,120 @@
+{
+  "info": {
+    "title": "",
+    "version": ""
+  },
+  "openapi": "3.0.3",
+  "paths": {
+    "/": {
+      "get": {
+        "responses": {
+          "200": {
+            "content": {
+              "application/json": {
+                "schema": {
+                  "example": 1,
+                  "type": "integer"
+                }
+              }
+            },
+            "description": ""
+          },
+          "201": {
+            "content": {
+              "application/json": {
+                "schema": {
+                  "example": 1.1,
+                  "type": "number"
+                }
+              }
+            },
+            "description": ""
+          },
+          "202": {
+            "content": {
+              "application/json": {
+                "schema": {
+                  "additionalProperties": false,
+                  "properties": {
+                    "negativeFloat": {
```

Comment:
Please, align the order of fields in OpenAPI according to the fields order in JSight file. Otherwise it is very difficult to manually check the test.

Reply:
fixed